cpp/data/system_updates/8.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -o errexit
```

Comment:
maybe add -o nounset just for consistency with other scripts? (+pipefail?)

Reply:
Done